api/types/loadrestrictions.go

```diff
@@ -3,7 +3,7 @@
 
 package types
 
-// Restrictions on what things can be referred to
```

Comment:
This doesn't look quite right. "Restrictions", in this context, was not referring to the name of the field.